NEWS

```diff
@@ -12,6 +12,11 @@
   which have empty repository scopes (gh#SUSE/machinery#1268)
 * Inspection of unmanaged-files is now also using the faster machinery-helper
   when the files are extracted
+* Introduce format version 5 which adds the "environment" scope (see
+  https://github.com/SUSE/machinery/blob/master/docs/System-Description-Format.md#version-5)
+  This scope is a hidden scope which is used internally by Machinery
```

Comment:
I would also add a period at the end here, as this entry is not using the short form where we leave out the period, but consists of full sentences.